packages/network/lib/agent.ts

```diff
@@ -8,11 +8,89 @@ import url from 'url'
 import { createRetryingSocket, getAddress } from './connect'
 import { lenientOptions } from './http-utils'
 import { ClientCertificateStore } from './client-certificates'
+import { promises as fs } from 'fs'
+import tls from 'tls'
 
 const debug = debugModule('cypress:network:agent')
 const CRLF = '\r\n'
 const statusCodeRe = /^HTTP\/1.[01] (\d*)/
 
+// Load and cache the contents of process.env.npm_config_cafile so that we don't have to read it
```

Comment:
These new functions feel self-contained enough to move to a `ca.ts` file instead of being at the top of `agent.ts`.

Reply:
Good call. I moved them.